.github/workflows/build.yml

```diff
@@ -46,6 +48,8 @@ jobs:
       - name: Setup MSVC
         uses: egor-tensin/vs-shell@9a932a62d05192eae18ca370155cf877eecc2202 # v2.1
         if: ${{ matrix.target_os == 'windows' }}
+      - name: install python3-requests
+        run: pip3 install requests
```

Comment:
Could you pin the hash and version?

Reply:
Same as https://github.com/NordSecurity/libtelio/pull/543#discussion_r1624164533